### PR TITLE
First PR for the branch!

### PR DESCRIPTION
**Pull request by AugustDG**
Just making the main app up to date, as this branch has something functional and stable!

This basically introduces a basic Admin Dashboard to view instructors and students (and manage them in the future)

### CODE REVIEW

File: lib/screens/admin/admin_home.dart [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+                alignment: Alignment.center,
+                icon: Icon(Icons.change_circle_rounded),
+                onPressed: () => setState(() => _isShowingStudents = !_isShowingStudents),
+              ),
+            ),
+            drawer: Drawer(
+              child: Column(
+                children: <Widget>[
+                  //Drawer's Header
+                  DrawerHeader(child: const SizedBox(width: 150), decoration: BoxDecoration(image: DecorationImage(image: Assets.images.followUpLogo01))),
```
Comment (Megasix) [inline lib/screens/admin/admin_home.dart:52]: you should show the logo according to the theme. light or dark.
image

**Review by Megasix — COMMENTED** [review]
Great work !!

File: lib/screens/admin/admin_home.dart (hunk 1 above)
Comment (AugustDG) [reply lib/screens/admin/admin_home.dart:57]: Ah ouais, mais je pensais mettre la photo de chaque auto-école la, donc c plus un placeholder, je le changerai tho!

Comment (AugustDG) [reply lib/screens/admin/admin_home.dart]: Now fixed!

File: lib/screens/admin/admin_home.dart [hunk 2]
```diff
@@ -55,7 +56,10 @@ class _AdminHomeState extends State<AdminHome> {
               child: Column(
                 children: <Widget>[
                   //Drawer's Header
-                  DrawerHeader(child: const SizedBox(width: 150), decoration: BoxDecoration(image: DecorationImage(image: Assets.images.followUpLogo01))),
+                  DrawerHeader(
+                      child: const SizedBox(width: 150),
+                      decoration:
+                          BoxDecoration(image: DecorationImage(image: Get.isDarkMode ? Assets.images.darkFollowUpLogo01 : Assets.images.followUpLogo01))),
```
Comment (Megasix) [inline lib/screens/admin/admin_home.dart:62]: The image should be reverse. It is mind fucking but when on light theme you use the dark image and vice versa

**Review by Megasix — APPROVED** [review]
Just the bool for the image, and the rest is just great

File: lib/screens/admin/admin_home.dart (hunk 2 above)
Comment (AugustDG) [reply lib/screens/admin/admin_home.dart:62]: Ahh...

**Pull request merged**